src/tcp.hpp

```diff
@@ -66,6 +66,10 @@ int tcp_read (fd_t s_, void *data_, size_t size_);
 //  Asserts that an internal error did not occur.  Does not assert
 //  on network errors such as reset or aborted connections.
 void tcp_assert_tuning_error (fd_t s_, int rc_);
+
+#if defined ZMQ_HAVE_WINDOWS
```

Comment:
perhaps it would be cleaner to move the ifdef inside the body of the function, and remove it from elsewhere, and just make it a no-op on non-windows?

Reply:
Fixed